### ISSUE TITLE
Ubuntu packages

### ISSUE THREAD

**Issue by forteller** [imprecision/franz-app#18]
I'd appreciate it very much if it was possible to install Franz on Ubuntu, not just run it. Could you please make a .deb or a snap for it? https://developer.ubuntu.com/en/snappy/build-apps/

Thanks!

**Comment by Mikaela** [comment]
Snap would be nice so it would be fast to update and no bureaucracy which is included with geting packages to Ubuntu repositories (even if I understand just .deb package is requested, not in repositories).

Debian package again would have advatance of also working on Debian and systems based on it.

**Comment by imprecision** [comment]
Yep, when we're closer to getting out beta this is very much on the todo list. Thanks for your patience on this in the mean time!
